Palace/Logging/TPPErrorLogger.swift

```diff
@@ -146,8 +146,6 @@ fileprivate let nullString = "null"
     guard Bundle.main.applicationEnvironment == .production else { return }
     
     #if FEATURE_CRASH_REPORTING
-    FirebaseApp.configure()
```

Comment:
We need to have it configured earlier. Moved to `applicationDidFinishLaunching`.